modules/configuration/php/configuration.class.inc

```diff
@@ -61,6 +61,12 @@ class Configuration extends \NDB_Form
             $scan_types[$val] = $val;
         }
 
+        $date_format = [
+            ''    => '',
+            'Ymd' => 'Ymd',
```

Comment:
Maybe it would be a good idea to show a more descriptive label for these on the frontend since it's a dropdown now?

Like `2015-12-29 (Ymd)` and `2015-12 (Ym)`?

(Date is arbitrary, just something that's unambiguous with the day of the month > 12.. I'm assuming m is numeric month, I haven't looked into it, but it would avoid that type of confusion for users to provide an example in the dropdown label..)